src/providers/token-provider.ts

```diff
@@ -404,6 +404,55 @@ export const WETH_POLYGON_MUMBAI = new Token(
   'Wrapped Ether'
 );
 
+// BNB chain Tokens
+export const BTC_BSC = new Token(
+  ChainId.BSC,
+  '0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c',
+  18,
+  'BTCB',
+  'Binance BTC'
+);
+
+export const BUSD_BSC = new Token(
+  ChainId.BSC,
+  '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56',
+  18,
+  'BUSD',
+  'BUSD'
+);
+
+export const DAI_BSC = new Token(
+  ChainId.BSC,
+  '0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3',
+  18,
+  'DAI',
+  'DAI'
+);
+
+export const ETH_BSC = new Token(
+  ChainId.BSC,
+  '0x2170Ed0880ac9A755fd29B2688956BD959F933F8',
+  18,
+  'ETH',
+  'ETH'
+);
+
+export const USDC_BSC = new Token(
+  ChainId.BSC,
+  '0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d',
+  18,
```

Comment:
triple checking this shouldnt be 6 like other chains? ik the contract says 18 but just flagging

Reply:
yeah looks to be 18, it is weird though